src/App.css

```diff
@@ -0,0 +1,84 @@
+:root {
```

Comment:
This is good, but a better approach to achieve the same thing is to have two classes as follows:
```
.light {
 --main-color: #ef9b28;
 --second-color: rgba(49, 89, 81, 0.9);
}

.dark {
 --main-color: #121826;
 --second-color: #060F20;
}
```

and give these classes to a parent container, and the children will have access to these css variables.
but we use the same name, and the values changes according to the class attached to the parent. if it is dark, we will have access to the dark theme, and if it is light, we will have access to the light theme without changing the names of the variables.